samza-core/src/test/java/org/apache/samza/config/TestJobConfig.java

```diff
@@ -610,11 +610,6 @@ public void testGetTaskOptsAutosizingDisabled() {
     ShellCommandConfig shellCommandConfig =
         new ShellCommandConfig(new MapConfig(ImmutableMap.of(JobConfig.JOB_AUTOSIZING_ENABLED, "false")));
     assertEquals(Option.empty(), shellCommandConfig.getTaskOpts());
-
-    String taskOpts = "-Dproperty=value";
-    shellCommandConfig = new ShellCommandConfig(new MapConfig(
-        ImmutableMap.of(ShellCommandConfig.TASK_JVM_OPTS(), taskOpts, JobConfig.JOB_AUTOSIZING_ENABLED, "false")));
-    assertEquals(Option.apply(taskOpts), shellCommandConfig.getTaskOpts());
```

Comment:
I don't think this should be removed. It is actually a different test case (i.e. autosizing is disabled).